services/device-registry-base/src/main/java/org/eclipse/hono/service/DeviceRegistryMetrics.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2022 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.eclipse.hono.service;
```

Comment:
we will need to rename this package as it conflicts with the package in the hono-service-base module.
How about *org.eclipse.hono.deviceregistry.metrics*